### PR TITLE
Update scripts to follow scripting conventions

### PR DESCRIPTION
**Pull request by 5abeel**

### CODE REVIEW

**Comment by vinintel** [comment]
Is that script changes verified on fedora/rocky linux ?

File: env_setup_acc.sh [hunk 1]
```diff
@@ -1,42 +1,74 @@
...
-       then
-           NUM_THREADS=$((NUM_CORES / 4))
-           NUM_THREADS=-j$NUM_THREADS
-       else
-           NUM_THREADS=-j${NUM_CORES}
-       fi
+# Check if NUM_CORES and NUM_THREADS are already set
+if [[ -n $NUM_CORES && -n $NUM_THREADS ]]; then
+    echo "NUM_CORES and NUM_THREADS already set."
+    exit 0
```
Comment (vinintel) [inline env_setup_acc.sh:38]: shouldn't it exit the shell if you run env script two times

File: install_dep_packages_acc.sh [hunk 2]
```diff
@@ -1,17 +1,11 @@
-# Copyright (c) 2021 Intel Corporation.
+#!/bin/bash
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at:
+# Copyright 2023 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-only
```
Comment (vinintel) [inline install_dep_packages_acc.sh:4]: Any reason for moving to SPDX-License from  Apache License ?

File: env_setup_acc.sh (hunk 1 above)
Comment (5abeel) [reply env_setup_acc.sh]: It shouldnt. This needs to be idempotent. I have removed this section

File: install_dep_packages_acc.sh (hunk 2 above)
Comment (5abeel) [reply install_dep_packages_acc.sh:4]: ipsec-recipe is not licensed under Apache as per this file: https://github.com/5abeel/ipsec-recipe/blob/main/LICENSE

File: env_setup_acc.sh [hunk 3]
```diff
@@ -1,42 +1,68 @@
...
+# Copyright 2023 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-only
+#
+# Script to setup environment variables facilitating IPsec-Recipe compilation
+#
+
+set -e
+
+# Define directories
+DEPS_INSTALL_DIR="PLUGIN_DEP_INSTALL"
```
Comment (ffoulkes) [inline env_setup_acc.sh:12]: 1. Variables ending in `_DIR` are usually directory paths, not bare directory names.
2. I also see no reason for assigning the bare directory name to a variable, using that to define a second variable, and never using the first variable again.

```bash
DEPS_INSTALL=$PWD/PLUGIN_DEP_INSTALL
SRC_DIR=$PWD/PLUGIN_DEP_DIR

mkdir -p ${DEPS_INSTALL}
mkdir -p ${SRC_DIR}
```
Just define one variable for each and use it throughout the script.

**Pull request merged**